db/migrate/20090714132449_add_audits_table.rb

```diff
@@ -12,6 +12,7 @@ def self.up
       t.column :comment, :string
       t.column :auditable_parent_id, :integer
       t.column :auditable_parent_type, :string
+      t.column :request_uuid, :string
```

Comment:
given we add the column in separate migration, why to edit this old file as well: I would prefer leaving old migrations untouched, and introduce the changes just in new files, because it causes confusion, as on can't be sure than, which version was used for actually migrating the data.